shared/haml/user_header.haml

```diff
@@ -135,20 +135,22 @@
       });
     });
 
-    function hideCreateOptions() {
-      $('.create_options').slideUp();
-      $('.create_menu_arrow_down').show();
-      $('.create_menu_arrow_up').hide();
-      $(document).off('click', hideCreateOptions);
+    function hideCreateOptions(event) {
+      if ((event.type === "keypress" && event.which === 13) || (event.type === "click")) {
+        $('.create_options').slideUp();
+        $('.create_menu_arrow_down').show();
+        $('.create_menu_arrow_up').hide();
+        $(document).off('click', hideCreateOptions);
+      }
     }
     $(document).ready(function() {
-      $('.create_menu').click(function (e) {
+      $('.create_menu').click(function(e) {
         if ($('.create_options').is(':hidden')) {
           e.stopPropagation();
           $('.create_options').slideDown();
           $('.create_menu_arrow_down').hide();
           $('.create_menu_arrow_up').show();
-          $(document).on('click', hideCreateOptions);
+          $(document).on('keypress click', (e) => hideCreateOptions(e));
```

Comment:
question for my own understanding - what's the difference between the syntax `(e) => hideCreateOptions(e)` and `hideCreateOptions` ?